### ISSUE TITLE
Serenity gradle report: Only one Feature is displayed correctly in features view

### ISSUE THREAD

**Issue by pbaeck** [serenity-bdd/serenity-core#2026]
Environment:

Windows 10
Java 8
Gradle 5.6
Serenity versions:
net.serenity-bdd:serenity-screenplay:2.0.52
net.serenity-bdd:serenity-cucumber4:1.0.21
net.serenity-bdd:serenity-gradle-plugin:2.2.0

Problem:
When I execute tests and the report is generated it shows all tests on the overview page correctly but when I switch to "Features" only one Feature is displayed correctly and the others not.

Overview:
![image](https://user-images.githubusercontent.com/20397225/79203218-fdd4b500-7e3a-11ea-8ba0-6c16196fce93.png)

Features View:
![image](https://user-images.githubusercontent.com/20397225/79203275-18a72980-7e3b-11ea-87a4-7829931ff46c.png)

I know we are using old versions of serenity due to performance problems with new versions, but can you give me a hint where I can place a breakpoint to figure out why we are having this problem?

**Comment by wakaleo** [comment]
It’s very hard to support older versions - what performance issue are you seeing with 2.2.1?

**Comment by pbaeck** [comment]
For about 2 weeks a colleague wanted to update to 2.1.8 and noticed that the tests performed much slower. As we have to prioritize our work because of the current situation we rolled back the version and worked on other issues. I can try to update the versions locally and will give you feedback if the report performs the same with the acutal versions.

**Comment by wakaleo** [comment]
That would be useful. It isn’t possible to do bug fixes for older versions.

**Comment by pbaeck** [comment]
Ok, I managed to upgrade the serenity versions today and the problem keeps the same.

The versions I use are now:
net.serenity-bdd:serenity-screenplay:2.2.0
net.serenity-bdd:serenity-cucumber4:2.1.2
net.serenity-bdd:serenity-gradle-plugin:2.2.0

In Features View of the serenity report is only one feature displayed correctly (Number of scenarios/Result/ID are not null)

Can you give me a hint where I should set a breakpoint to get to know what's missing?

**Comment by wakaleo** [comment]
For version 2.2.2 you need to use serenity-cucumber5. The error you describe might be related to the name of the feature or scenarios - could you post the feature files here?

**Comment by pbaeck** [comment]
I created a test project so that I can share it with you:

[serenity-test.zip](https://github.com/serenity-bdd/serenity-core/files/4486336/serenity-test.zip)

I initially added 2 feature files (TestSomething.feature&TestAnything.feature), then I run the test and checked the report and both feature files were also in the Features View in the Serenity-Report correctly displayed.

Then I added TestMe.feature and run the tests again. The feature is displayed in the Overview but not in the Feature View. I already executed gradle clean and clearReports but the problem still occurs. 

Do you have any suggestion?

**Comment by wakaleo** [comment]
The versions are not aligned. Try with the following build.gradle file:

```
buildscript {
    repositories {
        //mavenLocal()
    }
    dependencies {
        classpath "net.serenity-bdd:serenity-gradle-plugin:2.2.0"
    }
}

plugins {
    id 'java'
    id 'idea'
}

repositories {
    //mavenLocal()
}

apply plugin: 'net.serenity-bdd.aggregator'

sourceCompatibility = 1.8

repositories {
    mavenCentral()
}


dependencies {

    testRuntimeOnly("org.junit.vintage:junit-vintage-engine:5.6.0")

    testCompile "net.serenity-bdd:serenity-core:2.2.0",
            "net.serenity-bdd:serenity-cucumber5:2.2.0"

}


test {

    //always execute
    outputs.upToDateWhen { false }

    useJUnitPlatform()
    testLogging {
        events "passed", "skipped", "failed"
    }

    //pass system properties to test task
    systemProperties System.getProperties()

    finalizedBy aggregate
}
```

(Also, "Narrative:" is not a Gherkin keyword)

**Comment by pbaeck** [comment]
Thank you for the feedback, I changed the build.gradle and removed the Narrative text from the feature files.

But I have the ongoing problem that the report doesn't show added features in Features View when I execute it locally. 

I will describe stepwise what I did:

I use IntelliJ for test execution

1. I run all tests and open report 
    _Result: Every feature is displayed correctly in overview and features-view_
2. I add a new Feature file (Copy&Paste and Modified)
[TestAnything2.zip](https://github.com/serenity-bdd/serenity-core/files/4521497/TestAnything2.zip)

3. I execute "gradlew clean"
4. I run all tests and open report 
    _Result: Every feature is displayed correctly in overview but the new added Feature is missing in features-view_

I also tried to open it in a different browser (chrome vs. firefox) and deleted .gradle and .idea folder but the view and report keep the same, which cache do I have to clear?
